platform/platform-logging/src/test/java/org/codice/ddf/logging/platform/LoggingServiceBeanTest.java

```diff
@@ -0,0 +1,258 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.logging.platform;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Properties;
+
+import javax.management.MBeanServer;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.ops4j.pax.logging.spi.PaxLevel;
+import org.ops4j.pax.logging.spi.PaxLoggingEvent;
+
+@RunWith(MockitoJUnitRunner.class)
+public class LoggingServiceBeanTest {
+
+    private static final String BUNDLE_NAME = "bundle.name";
+
+    private static final String BUNDLE_VERSION = "bundle.version";
+
+    private static final int DEFAULT_LOG_EVENTS_LIMIT = 500;
+    
+    private static final int MAX_LOG_EVENTS_LIMIT = 5000;
+
+    @Mock
+    private MBeanServer mockMBeanServer;
+
+    @Test
+    public void testRetrieveLogEvents() {
+        // Setup
+        List<PaxLoggingEvent> mockPaxLoggingEvents = getMockPaxLoggingEventsTimestampOrderSmallestToLargest(3);
+        List<LogEvent> expectedLogEvents = getExpectedLogEvents(mockPaxLoggingEvents);
+        LoggingServiceBean loggingServiceBean = new LoggingServiceBean();
+        loggingServiceBean.init();
+        appendLogs(loggingServiceBean, mockPaxLoggingEvents);
+
+        // Perform Test
+        List<LogEvent> actualLogEvents = loggingServiceBean.retrieveLogEvents();
+
+        // Verify
+        assertThat(actualLogEvents,
+                contains(expectedLogEvents.toArray(new LogEvent[expectedLogEvents.size()])));
+    }
+
+    @Test
+    public void testResizeMaxLogEventsFrom500To1() {
```

Comment:
Instead of using numbers in the test names, should the names describe what we're trying to test instead, e.g., `testReduceMaxLogEventsKeepsAppropriateEvents()`, or something like that?